Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -198,12 +202,29 @@ public struct PackageBuilder {
         guard let pkgConfig = manifest.package.pkgConfig else { return nil }
         return RelativePath(pkgConfig)
     }
+
+    // FIXME: Should this be an extension on FileSystem and placed in Utility?
+    /// Returns path to all the items in a directory.
+    func directoryContents(_ path: AbsolutePath) throws -> [AbsolutePath] {
+        return try fileSystem.getDirectoryContents(path).map { path.appending(component: $0) }
+    }
+
+    // FIXME: Optimize to probably a lazy iterator and find correct home for this method.
+    /// Returns path to all the items in a directory tree.
```

Comment:
Was this just for convenience? Why go this route instead of migrating `walk` to `FileSystem`?

My perspective is that `walk` is somewhat uncommon, but very useful functionality to have, and so it makes sense to have in `Basic`. If we have it, not sure why we shouldn't use it here. I do think we should use `getDirectoryContents` directly except in the places we actually need the recursive walk (i.e., don't use walk just to get an individual directory contents).